R/jit-ops.R

```diff
@@ -0,0 +1,45 @@
+#' @export
```

Comment:
Last thing! I think we need to add at least some documentation to that object. Basically a title, description and a few examples on usage would be nice. Also add it to the pkgdown index in: https://github.com/mlverse/torch/blob/main/_pkgdown.yml

Reply:
done :-)
As to the pkgdown index, I think it should already get included (`starts_with("jit")`), right?